src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -13,77 +13,47 @@
  * Used to handle the data validation aspect e.g. validate emails, names, etc.
  */
 public class FieldValidator {
-    // ////////////////////////////////////////////////////////////////////////
-    // ////////////////// Generic types ///////////////////////////////////////
-    // ////////////////////////////////////////////////////////////////////////
+
+    /////////////////
+    // FIELD TYPES //
+    /////////////////
+
+    // name-related
+    public static final String PERSON_NAME_FIELD_NAME = "a person name";
```

Comment:
Sometimes the field name has the article "a/an", sometimes it doesn't. Let's aim to fix this one as well while we're at it.